src/assets/stylesheets/create-object-dialog.scss

```diff
@@ -0,0 +1,64 @@
+@import 'shared';
+
+:local(.smallButton) {
+  padding-right: 5%;
```

Comment:
These % based paddings seem odd. The buttons are of fixed sizes, why would their padding not be?

Reply:
Switched to using `em` values, though the style in general is not looking so good as I vary the screen sizes. Hoping to get some help with this.